src/components/login/profile.tsx

```diff
@@ -15,14 +18,17 @@ const Profile = () => {
     await authService.logout();
     setUser(null);
   };
+
+  const back = () => {
+    const lastUrl = sessionStorage.getItem("lastUrl");
+    lastUrl ? (window.location.href = lastUrl) : navigate("/");
```

Comment:
```suggestion
    setTimeout(() =>
      lastUrl ? (window.location.href = lastUrl) : navigate("/")
    );
```

Similar to register.tsx, to avoid  that "Cannot update a component while rendering a different component" error in the console.